megamek/src/megamek/common/BoardIndex.java

```diff
@@ -0,0 +1,154 @@
+package megamek.common;
+
+        /*
+        * MegaMek - Copyright (C) 2000,2001,2002,2003,2004 Ben Mazur (bmazur@sev.org)
+        * Copyright © 2013 Edward Cullen (eddy@obsessedcomputers.co.uk)
+        *
+        *  This program is free software; you can redistribute it and/or modify it
+        *  under the terms of the GNU General Public License as published by the Free
+        *  Software Foundation; either version 2 of the License, or (at your option)
+        *  any later version.
+        *
+        *  This program is distributed in the hope that it will be useful, but
+        *  WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+        *  or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+        *  for more details.
+        */
```

Comment:
This should go above the package, and further not at this level of indent.
A modern example of how this would be done properly is in AbstractIcon.